portfolio/src/main/webapp/about.html

```diff
@@ -1,48 +1,55 @@
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+	<script>
+		loadSelector("#nav", "navbar.html");
+	</script>
+  <script>
+		loadSelector("#social", "socials.html");
+	</script>
+</head>
 
-  </head>
-  <body>
-     
-      <h1 class="jumbotron"> <center> About Me</center> </h1>
-    <nav>
-    <ul>
-  <li><a href="index.html">Home</a></li>
-  <li><a href="about.html">About</a></li>
-  <li><a href="gallery.html">Gallery</a></li>
-  <li><a href="poems.html"> Favorite Poems</a></li>
-  <li><a href="topten.html"> My Top Ten Anime </a></li>
-    </ul>
-    </nav>
-      
-       <div class="container">
+<body>
+	<h1 class="jumbotron">
+		<center> About Me</center>
+	</h1>
+	<div id="nav"></div>
+	<div class="container">
+		<p>
+			<h3>
+				Some information on me: I am Nigerian-American -- born to Nigerian immigrant parents and I wholly
+				identify with both aspects. I currently study Computer Science at Xavier University of Louisiana, a
+				Private Catholic HBCU in New Orleans, Louisiana. Some of my summer goals include learning how to do a
+				backflip, getting in the best shape of my life, and learning Japanese (also on Duolingo).
 
+				Miscellaneous facts:
 
-      <p>
-        <h3>
-        Some information on me: I am Nigerian-American -- born to Nigerian immigrant parents and I wholly identify with both aspects. I currently study Computer Science at Xavier University of Louisiana, a Private Catholic HBCU in New Orleans, Louisiana. Some of my summer goals include learning how to do a backflip, getting in the best shape of my life, and learning Japanese (also on Duolingo).
-        
-        Miscellaneous facts:
+				I lived in Dhaka, Bangladesh
+				I speak 3 languages - English, Igbo, and Spanish
+      <br>
+      <br>
+      <br>
+			</h3>
+		</p>
+	</div>
+  <div class="container">
+		<br>
+		<img src="media/child.JPG" alt="Young tyke" class="img-circle" style="width:410px;height:410px;">
+	</div>
+  <div id="social"></div>
+</body>
 
-        I lived in Dhaka, Bangladesh
-        I speak 3 languages - English, Igbo, and Spanish
-
-        </h3>
-     </p>
-
-    </div>
-    <!-- Javascript for Bootstrap -->
-  <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
-  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
-  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
-  </body>
-</html>
+</html>
```

Comment:
Lets add a new line to the end of all the files
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline